angular-universal-ssr/host-app/server.js

```diff
@@ -3,5 +3,5 @@
 const mainModule = __non_webpack_require__.main;
 const moduleFilename = (mainModule && mainModule.filename) || '';
 if (moduleFilename === __filename || moduleFilename.includes('iisnode')) {
-  import('./server-bootstrap').then(x => x.run());
+  import('./server-bootstrap.ts').then(x => x.run());
```

Comment:
After clicking on `Federation lazy module`(http://localhost:4000/federation) get error
<img width="1661" alt="Screenshot 2023-01-12 at 19 31 19" src="https://user-images.githubusercontent.com/55971575/212139157-0134b3e8-114a-4dde-b4db-45ea7ed3abce.png">

Also please take a look

Reply:
Have you made yarn install for both apps and root?
![image](https://user-images.githubusercontent.com/26062537/212140568-791101d9-9d70-4d21-bfd1-914d8f6c5d59.png)

Reply:
Mine local works, that's why I'm asking
![image](https://user-images.githubusercontent.com/26062537/212140716-08670509-ddbf-4c70-9149-4a76d5040122.png)